projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -1864,6 +1864,23 @@ describe('IgxGrid Component Tests #grid', () => {
 
             expect(grid.columnList.get(0).width).toBe('50%');
         });
+
+        it('should correctly autosize column headers when the grid container has no data and is initially hidden and then shown', async () => {
+            const fix = TestBed.createComponent(IgxGridColumnHeaderAutoSizeComponent);
+            const grid = fix.componentInstance.grid;
+
+            //waiting for reqeustAnimationFrame to finish
+            await wait(17);
```

Comment:
It this case even just await wait() would be enough, to wait for the next tick. It is still fine as it is though.

Reply:
@IMinchev64